bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/GuideGroup.java

```diff
@@ -369,7 +364,7 @@ public boolean containsTarget(SPTarget target) {
 
     @Override
     public GuideGroup removeTarget(final SPTarget target) {
-        final ImList<GuideProbeTargets> updated = guideTargets.map(gpt -> gpt.removeTargetSelectPrimary(target));
```

Comment:
This version of `GuideGroup` is going away anyway in favor of the implementation based on the new model, but is this what you meant?  It looks like the semantics change there?